strawberry/tools/merge_types.py

```diff
@@ -0,0 +1,22 @@
+from typing import Tuple, Type
+
+import strawberry
+
+
+def merge_types(types: Tuple[Type]) -> Type:
+    """Merge multiple Strawberry types into one
+
+    For example, given two queries `A` and `B`, one can merge them as follows:
+
+        merge_types((B, A))
+
+    This is essentially the same as:
+
+        class Query(B, A):
+            ...
+    """
+
+    if not types:
+        raise ValueError("Can't merge types if none are supplied")
+
+    return strawberry.type(type("MegaType", types, {}))
```

Comment:
`merge_types` should probably accept a `name` argument so that you can customise the type name. I do like "MegaType" though so maybe that could be the default?

Reply:
Added a name arg, see https://github.com/strawberry-graphql/strawberry/pull/1273/commits/765c92313411c81cb03b056cb11af127424bac65